templates/vars/vnet/aat.json

```diff
@@ -66,6 +66,22 @@
           "service": "Microsoft.Storage"
         }
       ]
-    }
+    },
+    "routetableRoutes": {
+      "value": [
+          {
+          "name": "BastionMgmtVnet",
```

Comment:
is this actually needed? since you've got the hub route in and you're sending all traffic there?

Reply:
this seems an issue?

Reply:
Removed BastionMgmtVnet reference(s)